lib/rubocop/cop/rspec/describe_method.rb

```diff
@@ -23,20 +23,28 @@ class DescribeMethod < Base
         MSG = 'The second argument to describe should be the method ' \
               "being tested. '#instance' or '.class'."
 
-        # @!method second_argument(node)
-        def_node_matcher :second_argument, <<~PATTERN
+        # @!method second_string_literal_argument(node)
+        def_node_matcher :second_string_literal_argument, <<~PATTERN
           (block
-            (send #rspec? :describe _first_argument $(str _) ...) ...
-          )
+            (send #rspec? :describe _first_argument ${str dstr} ...)
+          ...)
+        PATTERN
+
+        # @!method method_name?(node)
+        def_node_matcher :method_name?, <<~PATTERN
```

Comment:
What advantages does this node matcher have over a regular method ? E.g.
```
def method_name?(str)
  str.start_with?('.', '#')
end
```

Reply:
The argument is a node, and if it’s a dstr, I’m uncertain if it has a ‘strvalue’ similar to the str type. But it’s worth checking out.

Reply:
This worked:
```ruby
        def method_name?(node)
          if node.str_type?
            method_name_prefix?(node.str_content)
          else # dstr
            method_name_prefix?(node.children.first.str_content)
          end
        end
```
Node pattern seems more concise.